generator/postgres/postgres_generator.go

```diff
@@ -66,7 +74,7 @@ func GenerateDSN(dsn, schema, destDir string, templates ...template.Template) er
 		return fmt.Errorf("failed to get '%s' schema metadata: %w", schema, err)
 	}
 
-	dirPath := filepath.Join(destDir, cfg.Database)
+	dirPath := filepath.Join(destDir, dbName)
```

Comment:
Hmm, I don’t see much value in passing two separate strings to functions only to have them joined within the function. It seems better to join them into `destDir` in the caller instead, so `dbName` can be removed from this method.